bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/internal/findandreplace/overlay/FindReplaceOverlay.java

```diff
@@ -1061,4 +1077,15 @@ private void setContentAssistsEnablement(boolean enable) {
 	private void updateContentAssistAvailability() {
 		setContentAssistsEnablement(findReplaceLogic.isAvailableAndActive(SearchOptions.REGEX));
 	}
+
+	private void decorate() {
+		searchBar.addModifyListener(event -> {
+			if (regexSearchButton.getSelection()) {
+				searchDecoration.decorateA(controldecoration, getFindString());
+			} else
+				controldecoration.hide();
```

Comment:
Add brackets to `else`
```suggestion
			} else {
				controldecoration.hide();
			}
```

Reply:
done